community.md

```diff
@@ -8,11 +8,27 @@ permalink: /community/
 
 The Ballerina community has been bootstrapped by <a href="https://wso2.com">WSO2</a> via sponsorship of the design & implementation of the Ballerina platform. Ballerina is a fully open source project with all source code licensed under the <a href="https://www.apache.org/licenses/LICENSE-2.0">Apache License, Version 2.0</a> and the specifications licensed under the <a href="https://creativecommons.org/licenses/by-nd/4.0">Creative Commons Attribution-NoDerivatives 4.0 International</a> license.
 
-We warmly welcome any and all contributors to the community to help establish Ballerina as a truly community-owned resource!
+We warmly welcome all contributors to the community to help establish Ballerina as a truly community-owned resource!
 
-## Looking for help?
+- [Try out Ballerina](#try-out-ballerina)
+- [Seek for help](#seek-for-help)
+- [Report issues](#report-issues)
+- [Help us grow](#help-us-grow)
+- [Contribute to Ballerina](#contribute-to-ballerina)
 
-We’re happy to help! Come engage with us on any channel that works for you:
+## Try out Ballerina
+
+Use the below resources to learn Ballerina and try out writing Ballerina code on your own.
+- [Quick Tour](https://ballerina.io/learn/quick-tour)
+- [Ballerina By Examples](https://ballerina.io/learn/by-example/)
+- [How-To Guides](https://ballerina.io/learn/)
+- [The Standard Library](https://ballerina.io/learn/api-docs/ballerina/)
+
+>**Tip:** Do not forget to share the wealth by publishing your Ballerina module on <a href="https://central.ballerina.io">Ballerina Central</a> so that the whole community can benefit from your work. 

```

Comment:
This content seems repeating in the `Help us grow` section as well.

Reply:
Removed this Tip.